frontend/src/main/web/lib/actions/GlossaryActions.js

```diff
@@ -1,8 +1,9 @@
 import Dispatcher from '../dispatchers/GlossaryDispatcher';
 import {GlossaryActionTypes} from '../constants/ActionTypes';
 
-
 var Actions = {
+  NO_INFO_MESSAGE: 'No information available',
```

Comment:
What is l10n like for this stuff right now? Can this come from `messages.properties` or something?